src/foam/parse/DateParser.js

```diff
@@ -30,6 +35,42 @@ foam.CLASS({
     INVALID_DATE: new Date(NaN)
   },
 
+  static: [
+    {
+      name: 'getSymParser',
+      args: 'String symbolName',
+      type: 'foam.parse.Parser',
+      documentation: 'Static convenience method to get a symbol parser from the DateParser grammar. Example: foam.parse.DateParser.getSymParser("ddmmyyyy-sep")',
+      code: function(symbolName) {
+        return foam.parse.DateParser.create().grammar_.getSymParser(symbolName);
+      }
+    },
+    {
+      name: 'getRawSymbol',
```

Comment:
Why do you need this? Isn't it better if it already has the action on it so that it parses it to a date, since we need to do that eventually and if we do it now then we don't have to reparse it again.